Porting/perldocstyle.pod

```diff
@@ -0,0 +1,1094 @@
+features altogether.
+
+More thorough reference materials or explanatory articles can include
+experimental features, but needs to clearly mark them as such, and not
+treat them with the same prominence as Perl's stable features. Using
+unstable features seldom coincides with best practices, and
+documentation that L<puts best practices first|/Lead with common cases
+and best practices> should reflect this.
+
+=head2 The documentation speaks with one voice
+
+Even though it comes from many hands and minds, criss-crossing through
+the many years of Perl's lifetime, the language's documentation should
+speak with a single, consistent voice. With few exceptions, the docs
+should avoid explicit first-person statements, or similar self-reference
+to any individual's contributor's philosophies or experiences.
+
+Perl did begin life as a deeply personal expression by a single
+individual, and this famously carried through the first revisions of its
+documentation as well. Today, Perl's community understands that the
+language's continued development and support comes from many people
+working in concert, rather than any one person's vision or effort. Its
+documentation should not pretend otherwise.
+
+The documentation should, however, carry forward the best tradition that
+Larry Wall set forth in the language's earliest days: Write both
+economically and with a humble, subtle wit, resulting in a technical
+manual that mixes concision with a friendly approachability. It avoids
+the dryness that one might expect from technical documentation, while
+not leaning so hard into overt comedy as to distract and confuse from
+the nonetheless-technical topics at hand.
+
+Like the best written works, Perl's documentation has a soul. Get
+familiar with it as a reader to internalize its voice, and then find
+your own way to express it in your own contributions. Writing clearly,
+succinctly, and with knowledge of your audience's expectations will get
+you most of the way there, in the meantime.
+
+Every line in the docs--whether English sentence or Perl statement --
+should serve the purpose of bringing understanding to the reader. Should
+a sentence exist mainly to make a wry joke that doesn't further the
+reader's knowledge of Perl, set it aside, and consider recasting it into
+a personal blog post or other article instead.
+
+Write with a light heart, and a miserly hand. 
+
+=head1 INDEX OF PREFERRED TERMS
+
+L<As noted above|/Choice of underlying style guide: CMOS>, this guide
+"inherits" all the preferred terms listed in the Chicago Manual of
+Style, 17th edition, and adds the following terms of particular interest
+to Perl documentation.
+
+=over
+
+=item built-in function
+
+Not "builtin".
+
+=item Darwin
+
+See L<macOS|/macOS>.
+
+=item macOS
+
+Use this term for Apple's operating system instead of "Mac OS X" or
+variants thereof.
+
+This term is also preferable to "Darwin", unless one needs to refer
+to macOS's Unix layer specifically.
```

Comment:
What about MacOS? Is that an acceptable alternative in the middle of a sentence, following the general English rule of names, ie proper nouns, beginning with an upper-case letter? What about capitalising at the beginning of a sentence?

Reply:
There should probably be a note about Windows, mentioning capitalising to avoid confusion between Windows the operating system and windows the things on the screen. Maybe "MS Windows" or even "Microsoft Windows". And a note about it never being "M$ Windows" or similar silliness?

Reply:
The name of the OS is "macOS" - "MacOS" is a little too close to "Mac OS" which is the name of their older OS. Yes, the names are terrible.

Reply:
Yep, it's "macOS", just like "iPhone", and the same (eye-rolling but trademarked) spelling rules apply.

As for Windows et al, I certainly expect this section to grow quite a bit, especially once the Great Work of reviewing and revising the present core docs gets underway. What's here is just a stub, based on the little bit of test revision I did for this project, plus the computer equipment I typed it all in on. :)